### ISSUE TITLE
question

### ISSUE THREAD

**Issue by riskeverything** [JJHu1993/realsense_ros#2]
Should I install librealsense firstly?Some problems happened.

collect2: error: ld returned 1 exit status
make[2]: *** [/home/robot/WorkSpace/catkin_ws/devel/lib/realsense_ros/realsense_ros] Error 1
make[1]: *** [realsense_ros/CMakeFiles/realsense_ros.dir/all] Error 2
make: *** [all] Error 2

**Comment by JJHu1993** [comment]
yes, since librealsense updated many times, you should git clone the legacy version and install.

**Comment by riskeverything** [comment]
OK,thanks very much.I had use it before.Recently,I reinstall my pc,and It doesn't work.So,should  I use the legacy version and install with the old "read me"?

**Comment by JJHu1993** [comment]
https://github.com/IntelRealSense/librealsense/tree/legacy

**Comment by riskeverything** [comment]
Recently，I use two cameras in my pc.but, it can only works little seconds.the topic is right,Then it will break....   
[ INFO] [1514460386.420596708]: rs::error was thrown when calling rs_wait_for_frames(device:0xbe18c8):
[ INFO] [1514460386.420647954]:     Timeout waiting for frames.
[realsense_ros-2] process has died [pid 30741, exit code 1, cmd /home/robot/WorkSpace/catkin_ws/devel/lib/realsense_ros/realsense_ros 617205001730 615205001368 __name:=realsense_ros __log:=/home/robot/.ros/log/43868ca0-ebc0-11e7-b314-6045cb646fe3/realsense_ros-2.log].

I use the example of librealsense.It has the same problems.But I remebered it work well.Maybe can you tell me some suggests?

**Comment by JJHu1993** [comment]
please confirm that the usb3.0 ports are power supplied.

**Comment by riskeverything** [comment]
Yes,I have resolve it.I use a hub powered.Anyway,thank.
